app/controllers/boundaries_controller.rb

```diff
@@ -28,20 +28,11 @@ def delete_boundary
         render json: boundary
     end
     def inside_by_name
-        boundary = Boundary.find_by_name(params[:name])
-        params = {
-            "point" => request.body.read,
-            "geojson" => boundary.coordinates,
-            "minx" => boundary.minx,
-            "maxx" => boundary.maxx,
-            "miny" => boundary.miny,
-            "maxy" => boundary.maxy
-        }
-        if boundary
-            is_inside = InsidePolygon.point_in_poly(params)
-            render json: {is_inside: is_inside}, status: :ok
+        is_inside = InsidePolygon.point_in_poly_processing(request.body.read, params[:name])
```

Comment:
I would setup the params for the request to be something like:

`lat`, `lng`, `boundary_name`, `boundary_id`

I generally try to minimize the ability to pass whatever param past a controller